cli/src/commands/jobs.rs

```diff
@@ -94,6 +94,12 @@ pub async fn handle_history(api: &mut PhylumApi, matches: &clap::ArgMatches) ->
         .and_then(|v| Filter::from_str(v).ok());
 
     if let Some(matches) = matches.subcommand_matches("project") {
+        if pretty_print {
```

Comment:
I'm guessing this is to avoid causing problems in a piped scenario. But `print_user_warning!()` writes to stderr not stdout, so it's not actually an issue.

Reply:
Hm that's a fair point, I just didn't want to change any output that might be likely to be computer-read.